src/modules/bulmaModal.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable space-before-function-paren */
+/* eslint-disable func-names */
+/* eslint-disable camelcase */
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable import/prefer-default-export */
```

Comment:
It's OK to disable some rule for some special cases, using `// eslint-disable-next-line` or `// eslint-disable-line`.

But disabling so many rules for the whole file doesn't seem appropriate. You're basically skipping the rules provided.

Reply:
I've added "linebreak-style": "off" rule since it wasn't in the file. Thanks for point it out.

Please see summarized finds at the bottom of the page.